include/mimalloc-types.h

```diff
@@ -61,6 +61,11 @@ terms of the MIT license. A copy of the license can be found in the file
 #define MI_ENCODE_FREELIST  1
 #endif
 
+// Enable xmalloc()-like functionality: call abort() should a memory allocation
+// from system fail
+#if !defined(MI_XMALLOC)
+#define MI_XMALLOC 0
+#endif
```

Comment:
Can you remove this, we can just check MI_XMALLOC as being defined in `option.c`.  Thanks